tests/helper/helper_generic.go

```diff
@@ -119,6 +129,7 @@ func WatchNonRetCmdStdOut(cmdStr string, timeout time.Duration, check func(outpu
 		cmd = exec.Command(cmdName)
 	}
 	cmd.Stdout = &buf
+	cmd.Stderr = &errBuf
```

Comment:
@prietyc123 we never ever ignore the stderr in a subprocess execution. any error in the `odo watch` command would just stall the tests for 5mins

Reply:
cc @prietyc123